docs/version0.18.md

```diff
@@ -66,6 +66,9 @@ The Xdump "exit" agent is used in the implementation of `-XX:[+-]ExitOnOutOfMemo
 By caching the Latest User Defined Class Loader (LUDCL), Java applications that use deserialization extensively can see a performance improvement. This
 capability is controlled by the [-Dcom.ibm.enableClassCaching](dcomibmenableclasscaching.md) system property and is now enabled by default. This feature was disabled for the 0.17.0 release due to [issue #7332](https://github.com/eclipse/openj9/issues/7332) which has now been resolved.
 
+### Terabytes suffix support for -X and -XX options that take a size
```

Comment:
Don't forget to add this new topic to the table of contents above, using #terabytes-suffix-support-for-x-and-xx-options-that-take-a-size

Reply:
@pshipton: done change of table of contents, made commit and squashing !

Reply:
The commits aren't actually squashed.